vbpd/vbpd-noreflection/src/main/java/by/kirich1409/viewbindingdelegate/ViewBindingProperty.kt

```diff
@@ -139,25 +140,11 @@ public abstract class LifecycleViewBindingProperty<in R : Any, out T : ViewBindi
 
     private class ClearOnDestroyLifecycleObserver(
         private val property: LifecycleViewBindingProperty<*, *>
-    ) : DefaultLifecycleObserver {
-
-        override fun onCreate(owner: LifecycleOwner) {
-        }
-
-        override fun onStart(owner: LifecycleOwner) {
-        }
-
-        override fun onResume(owner: LifecycleOwner) {
-        }
-
-        override fun onPause(owner: LifecycleOwner) {
-        }
-
-        override fun onStop(owner: LifecycleOwner) {
-        }
+    ) : LifecycleObserver {
 
         @MainThread
-        override fun onDestroy(owner: LifecycleOwner) {
+        @OnLifecycleEvent(Lifecycle.Event.ON_DESTROY)
```

Comment:
It will not work without annotation processor. Remove annotation `@OnLifecycleEvent(Lifecycle.Event.ON_DESTROY)` and override all methods from LifecycleObserver with empty bode except `onDestroy`. it will fix the issue

Reply:
Don't forget to add reason of the strange implementation with reference to the bug and solutions

Reply:
LifecycleObserver is empty common interface and not have methods to override. You might mean FullLifecycleObserver, but it has package-internal access and is not available from here. (DefaultLifecycleObserver -> FullLifecycleObserver-> LifecycleObserver )